source/acquaman/AMGenericContinuousScanController.cpp

```diff
@@ -101,6 +101,9 @@ void AMGenericContinuousScanController::buildScanControllerImplementation()
 
 		foreach (AMRegionOfInterest *region, configuration_->regionsOfInterest()){
 
+			// This is sufficient to add a region of interest on all detectors as they should by synchronized via AMBeamline::synchronizeXRFDetectors.
+			detector->addRegionOfInterest(region);
```

Comment:
If the detector ROI information isn't changed during a scan, the detector ROIs should be the same as the configuration ROIs right? Is it possible to 'double add' regions of interest here?